server/chat-plugins/chat-monitor.ts

```diff
@@ -569,7 +569,7 @@ export const commands: ChatCommands = {
 			const output = `'${word}' was added to the ${list} list.`;
 			const upperStaff = Rooms.get('upperstaff');
 			if (upperStaff) upperStaff.add(output).update();
-			if (room.roomid !== 'upperstaff') return this.sendReply(output);
+			return this.sendReply(output);
```

Comment:
Same problem as before, this makes the output repetitive if used in Upper Staff.

Also, the `return` is unnecessary.